<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabInsert">
       <attribute name="title">
        <string>Insert</string>
       </attribute>
       <widget class="QLabel" name="labelTitle">
        <property name="geometry">
         <rect>
          <x>255</x>
          <y>20</y>
          <width>291</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Yu Gothic UI</family>
          <pointsize>24</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Quality Tracking</string>
        </property>
       </widget>
       <widget class="QWidget" name="formLayoutWidget">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>100</y>
          <width>291</width>
          <height>359</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="labelAssy">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Assembly Num:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEditAssy"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelPart">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Part Num:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEditPart"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="labelDesc">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Description:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="lineEditDesc"/>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="labelRepr">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Reporter:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLineEdit" name="lineEditReport"/>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="labelDate">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Date:</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QDateEdit" name="dateEdit">
           <property name="currentSection">
            <enum>QDateTimeEdit::DaySection</enum>
           </property>
           <property name="date">
            <date>
             <year>2021</year>
             <month>11</month>
             <day>11</day>
            </date>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="labelResp">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Responder:</string>
           </property>
          </widget>
         </item>
         <item row="10" column="1">
          <widget class="QComboBox" name="comboBoxResp">
           <property name="acceptDrops">
            <bool>false</bool>
           </property>
           <item>
            <property name="text">
             <string>ENGINEERING</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>PURCHASING</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>QUALITY</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="12" column="0">
          <widget class="QLabel" name="labelStat">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Status:</string>
           </property>
          </widget>
         </item>
         <item row="12" column="1">
          <widget class="QComboBox" name="comboBoxStat">
           <item>
            <property name="text">
             <string>NEW</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ONGOING</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>CLOSED</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="1">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="1">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="5" column="1">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="7" column="1">
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="9" column="1">
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="11" column="1">
          <spacer name="verticalSpacer_6">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="insertBtn">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>110</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Insert</string>
        </property>
       </widget>
       <widget class="QPushButton" name="updateBtn2">
        <property name="geometry">
         <rect>
          <x>590</x>
          <y>110</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Finish Update</string>
        </property>
       </widget>
       <widget class="QPushButton" name="updateBtn3">
        <property name="geometry">
         <rect>
          <x>590</x>
          <y>210</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Cancel Update</string>
        </property>
       </widget>
       <widget class="QProgressBar" name="progressBar">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>430</y>
          <width>301</width>
          <height>23</height>
         </rect>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
       <widget class="QPushButton" name="reportBtn">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>210</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Report</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditDir">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>380</y>
          <width>261</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="directoryBtn">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>320</y>
          <width>111</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Working Directory</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tabEdit">
       <attribute name="title">
        <string>Edit</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0" rowspan="4">
         <widget class="QTableView" name="tableView">
          <property name="minimumSize">
           <size>
            <width>531</width>
            <height>441</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>531</width>
            <height>441</height>
           </size>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhNone</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="loadBtn">
          <property name="text">
           <string>Load Database</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="deleteBtn">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="updateBtn">
          <property name="text">
           <string>Update</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="formatBtn">
          <property name="text">
           <string>Format</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabSearch">
       <attribute name="title">
        <string>Search</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>20</y>
          <width>160</width>
          <height>81</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>160</width>
          <height>81</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="labelCol">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Search Column:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxCol">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <item>
            <property name="text">
             <string>Assembly Num</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Part Num</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Date</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Responder</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Status</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>150</y>
          <width>161</width>
          <height>51</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>161</width>
          <height>51</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page">
         <widget class="QComboBox" name="comboBoxCond">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>20</y>
            <width>161</width>
            <height>22</height>
           </rect>
          </property>
          <item>
           <property name="text">
            <string>greater than or equal to</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>less than or equal to</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>equal to</string>
           </property>
          </item>
         </widget>
        </widget>
        <widget class="QWidget" name="page_2">
         <widget class="QComboBox" name="comboBoxCond2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>20</y>
            <width>161</width>
            <height>22</height>
           </rect>
          </property>
          <item>
           <property name="text">
            <string>equal to</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>not equal to</string>
           </property>
          </item>
         </widget>
        </widget>
       </widget>
       <widget class="QLabel" name="labelCond">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>130</y>
          <width>141</width>
          <height>21</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>141</width>
          <height>21</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Search Condition:</string>
        </property>
       </widget>
       <widget class="QStackedWidget" name="stackedWidget_2">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>260</y>
          <width>161</width>
          <height>51</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>161</width>
          <height>51</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page_3">
         <widget class="QLineEdit" name="lineEditNum">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>20</y>
            <width>113</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="page_4">
         <widget class="QDateEdit" name="dateEdit_2">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>20</y>
            <width>110</width>
            <height>22</height>
           </rect>
          </property>
          <property name="date">
           <date>
            <year>2021</year>
            <month>11</month>
            <day>11</day>
           </date>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="page_5">
         <widget class="QComboBox" name="comboBoxRe">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>20</y>
            <width>161</width>
            <height>22</height>
           </rect>
          </property>
          <item>
           <property name="text">
            <string>ENGINEERING</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>PURCHASING</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>QUALITY</string>
           </property>
          </item>
         </widget>
        </widget>
        <widget class="QWidget" name="page_6">
         <widget class="QComboBox" name="comboBoxSt">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>20</y>
            <width>161</width>
            <height>22</height>
           </rect>
          </property>
          <item>
           <property name="text">
            <string>NEW</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ONGOING</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>CLOSED</string>
           </property>
          </item>
         </widget>
        </widget>
       </widget>
       <widget class="QLabel" name="labelVar">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>235</y>
          <width>141</width>
          <height>21</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>141</width>
          <height>21</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Search Variable:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="searchBtn">
        <property name="geometry">
         <rect>
          <x>610</x>
          <y>390</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>93</width>
          <height>28</height>
         </size>
        </property>
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelSearch">
        <property name="geometry">
         <rect>
          <x>570</x>
          <y>330</y>
          <width>181</width>
          <height>41</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>181</width>
          <height>41</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="labelResults">
        <property name="geometry">
         <rect>
          <x>600</x>
          <y>430</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>121</width>
          <height>41</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Results:</string>
        </property>
       </widget>
       <widget class="QTableView" name="tableView_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>531</width>
          <height>441</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>531</width>
          <height>441</height>
         </size>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>formatBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>format()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>710</x>
     <y>402</y>
    </hint>
    <hint type="destinationlabel">
     <x>791</x>
     <y>358</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>updateBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>alter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>720</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>795</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>reportBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>report()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>558</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>792</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>insertBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>insert()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>550</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>792</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>directoryBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>SelectWorkingDirectory()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>573</x>
     <y>392</y>
    </hint>
    <hint type="destinationlabel">
     <x>792</x>
     <y>335</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>updateBtn2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>alter2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>690</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>794</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>load()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>696</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>792</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>delete()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>700</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>792</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>updateBtn3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>cancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>694</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel">
     <x>793</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>searchBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>search()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>711</x>
     <y>479</y>
    </hint>
    <hint type="destinationlabel">
     <x>792</x>
     <y>484</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxCol</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>index_change()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>736</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>793</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>insert()</slot>
  <slot>load()</slot>
  <slot>delete()</slot>
  <slot>alter()</slot>
  <slot>alter2()</slot>
  <slot>cancel()</slot>
  <slot>search()</slot>
  <slot>format()</slot>
  <slot>index_change()</slot>
  <slot>report()</slot>
  <slot>SelectWorkingDirectory()</slot>
  <slot>cell_clicked()</slot>
 </slots>
</ui>
